third-party/arcanum-worlds/odyssey-of-the-dragonlords/items.xml

```diff
@@ -37,4 +35,707 @@
+		</description>
+		<setters>
+			<set name="keywords">necklace</set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="slot">neck</set>
+			<set name="type">Wondrous Item</set>
+		</setters>
+	</element>
+	<element name="Amulet, Lunula Talisman" type="Magic Item" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_MAGIC_ITEM_AMULET_LUNULA_TALISMAN">
+		<description>
+			<p>This crescent moon-shaped golden amulet is engraved with beautiful filigree. While wearing this amulet, if your alignment is good, you have advantage on saving throws against spells and magical effects cast by aberrations and monstrosities.</p>
+		</description>
+		<setters>
+			<set name="keywords">necklace</set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="slot">neck</set>
+			<set name="type">Wondrous Item</set>
+		</setters>
+	</element>
+	<element name="Antikythera Device" type="Magic Item" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_MAGIC_ITEM_ANTIKYTHERA_DEVICE">
+		<description>
+			<p>The Antikythera is a complex construct that acts as a magical compass. It also has the power to mend damage to a ship: once per day it can magically repair 50 hit points of damage to a ship it is aboard.</p>
+			<p class="indent">It uses the constellations to guide its user to islands in the Cerulean Gulf or Forgotten Sea. Every island in these seas are associated with a specific constellation. The Antikythera must be used on dry land as the constant movement of waves makes it impossible to calibrate. When it is calibrated to the proper constellations, it will show a path that can be used to reach the island associated with a particular constellation.</p>
+		</description>
+		<setters>
+			<set name="keywords"></set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="attunement">true</set>
+			<set name="rarity">Legendary</set>
+		</setters>
+	</element>
+	<element name="Balm of Invulnerability" type="Magic Item" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_MAGIC_ITEM_BALM_OF_INVULNERABILITY">
+		<description>
+			<p>This ointment is made from the magical herbs that only grow where titan blood has been spilled. A character may apply this balm to their entire body to gain resistance to non-magical damage for 1 day. This uses the entire jar of balm.</p>
+		</description>
+		<setters>
+			<set name="keywords"></set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Very Rare</set>
+		</setters>
+	</element>
```

Comment:
with some items such as these balms or oils you can put the **stackable** setter in to allow the item to be stacked such as ammunition or potions (something you might have more than 1 of)
`<set name="stackable">true</set>`

(unless it's equippable/attunable)